AEPCampaign/Sources/rules/CampaignRulesCache.swift

```diff
@@ -46,18 +46,18 @@ struct CampaignRulesCache {
         }
     }
 
-    /// Gets the cached rules for the given rulesUrl
-    /// - Parameter rulesUrl: The rules url as a string to be used to get the right cached rules
-    /// - Returns: The `CachedRules` for the given rulesUrl
+    /// Gets the cached rules for the given rules url.
```

Comment:
Either use the provided var name `rulesUrl` or if describing simply use `rules URL`. Let's rewrite
`Gets the cached rules for the given rules URL.`